File: CVR/batch-3/python/fib.py




def find_sum(n):
    a,b = 0,1
    if(n<=0):
        return "Invalid input"
    if(n==1):
        return a
    if(n==2):
        return a+b
    sum = a+b
    count = 2
    while count !=n:
        next = a+b
        sum+=next
        a=b
        b=next
        count+=1
    return sum

print(find_sum(14))

File: CMR-EC/c1/Fib.java
public class Fib {
    public static void main(String[] args) {
        int n = 10;
        int p1= 0, p2 = 1;
        if (n <= 2) {
            if (n == 1) {
                System.out.println(0);
            }else if(n == 2){
                System.out.println(0+" "+1+" ");
            }
        }else{
            System.out.print(0+" "+1+" ");
            for(int i =3; i<=n ; i++){
                int next = p1+p2;
                System.out.print(next+" ");
                p1 = p2;
                p2 = next;
            }
        }
    }
}
